src/Symfony/Component/Templating/Tests/Loader/CacheLoaderTest.php

```diff
@@ -79,7 +80,7 @@ public function getSpecialTemplate()
         return 'Hello {{ name }}';
     }
 
-    public function load(TemplateReferenceInterface $template)
+    public function load(TemplateReferenceInterface $template): Storage|bool
```

Comment:
\Symfony\Component\Templating\Loader\LoaderInterface::load return type should be `Storage|false`

Reply:
fixed, thanks for the review!